src/modules/browser/speech-synthesis/preSelect/Languages.json

```diff
@@ -0,0 +1,75 @@
+{
+  "ar": "Arabic",
```

Comment:
For the languages we also have another list of TTS/ASR Languages in the config. I wonder if we need 2.

Reply:
Languages is a big problem even we have ISO languages code. People don't use it latest version ant it's chaos itself. so the reality is different provider have different code.

There are some solution:
1. For different modals/provider, use different languages file.
 benefit: easy to update, save dev energy, have more choices even some small languages
 issues: hard to link different part of app, user need to setting one by one.
2. use one languages file, manual maintenance different system's code
benefit: user set once. setting everywhere
issues: can only maintain intersection part, less languages. need manually maintain. Introducing additional complexity so may have bugs.